kDrive/UI/Controller/Menu/ParameterTableViewController.swift

```diff
@@ -203,17 +203,21 @@ class ParameterTableViewController: UITableViewController {
 
 extension ParameterTableViewController: DeleteAccountDelegate {
     func didCompleteDeleteAccount() {
```

Comment:
Here the service could be the `DeleteAccountDelegate` not a table view.

Reply:
I think Logging out / Switching account can indeed be refactored. A lot of code seems to be partially duplicated. However this should be done in an other PR.